contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1419,13 +1419,13 @@ The following example shows a more complex use of user-chosen elements and presc
 
 [[backslash-for-long-code-line]]
 ==== Backslash for long lines of code
-For long lines of code that you want to break up among multiple lines, use a backslash to show the line break. For example:
+For long lines of code that you want to break up among multiple lines, use a backslash to show the line break and indent all lines under the first with two spaces. For example:
 
 ----
 $ oc get endpoints --all-namespaces --template \
-    '{{ range .items }}{{ .metadata.namespace }}:{{ .metadata.name }} \
-    {{ range .subsets }}{{ range .addresses }}{{ .ip }} \
-    {{ end }}{{ end }}{{ "\n" }}{{ end }}' | awk '/ 172\.30\./ { print $1 }'
+  '{{ range .items }}{{ .metadata.namespace }}:{{ .metadata.name }} \
+  {{ range .subsets }}{{ range .addresses }}{{ .ip }} \
+  {{ end }}{{ end }}{{ "\n" }}{{ end }}' | awk '/ 172\.30\./ { print $1 }'
 ----
```

Comment:
```suggestion
----
....
```
Wondering if we should update and show the whole example code block, not just the middle?

Reply:
Good catch on the `....` lines. Doesn't seem to impact the output but I'll update it to be consistent.

What do you mean by "show the whole example code block, not just the middle"? I think it is the whole block that's shown here but I might be missing something

Reply:
Meaning adding like [source,terminal] or what have you.

Reply:
Oh - I see your earlier comment in my email from GH, I get what you mean now :nerd_face: I think it works either way but I can do that while I'm in there adding `.`s :slightly_smiling_face: